regions/web/footer.py

```diff
@@ -11,101 +11,145 @@ class Footer(Region):
-        self.find_element(*self._privacy_policy_locator).click()
-        sleep(1)
-        from pages.web.privacy import PrivacyPolicy
-        return PrivacyPolicy(self.driver)
-
-    def go_to_accessibility_statement(self):
-        """View the accessibility statement."""
-        self.find_element(*self._accessibility_statement_locator).click()
-        sleep(1)
-        from pages.web.accessibility import Accessibility
-        return Accessibility(self.driver)
-
-    def go_to_open_source(self):
-        """Open GitHub and view the OpenStax repositories."""
-        self.find_element(*self._opens_source_locator).click()
-        sleep(1)
-        from pages.github.home import GitHub
-        return GitHub(self.driver)
-
-    def go_to_contact_us(self):
-        """Go to the contact form."""
-        self.find_element(*self._contact_us_locator).click()
-        sleep(1)
-        from pages.web.contact import Contact
-        return Contact(self.driver)
-
-    def go_to_press(self):
-        """View the press page."""
-        self.find_element(*self._press_locator).click()
-        sleep(1)
-        from pages.web.press import Press
-        return Press(self.driver)
-
-    def go_to_newsletter(self):
-        """Go to the newsletter signup."""
-        self.find_element(*self._newsletter_locator).click()
-        sleep(1)
-        from pages.web.newsletter import NewsletterSignup
-        return NewsletterSignup(self.driver)
-
-    def go_to_facebook(self):
-        """Go to OpenStax's Facebook page."""
-        self.find_element(*self._facebook_locator).click()
-        sleep(1)
-        from pages.facebook.home import Facebook
-        return Facebook(self.driver)
-
-    def go_to_twitter(self):
-        """Go to OpenStax's Twitter page."""
-        self.find_element(*self._twitter_locator).click()
-        sleep(1)
-        from pages.twitter.home import Twitter
-        return Twitter(self.driver)
-
-    def go_to_linkedin(self):
-        """Go to OpenStax's LinkedIn company page."""
-        self.find_element(*self._linkedin_locator).click()
-        sleep(1)
-        from pages.linkedin.home import LinkedIn
-        return LinkedIn(self.driver)
+    @property
+    def statements(self):
+        """Return Statement region"""
+        return self.Statements(self)
+
+    @property
+    def content_links(self):
+        """Return ContentLinks region"""
+        return self.ContentLinks(self)
+
+    @property
+    def social_links(self):
+        """Return SocialLinks region"""
+        return self.SocialLinks(self)
+
+    class Statements(Region):
+        """Statements region."""
+
+        _hero_quote_locator = (By.CLASS_NAME, 'hero-quote')
+        _nonprofit_locator = (By.CSS_SELECTOR, "[role*='contentinfo'] p")
+        _copyright_locator = (By.CSS_SELECTOR, "[role*='contentinfo'] p")
+        _trademark_locator = (By.CSS_SELECTOR, 'ap-html')
+
+        def hero_statement(self):
+            """Returns the hero banner statement."""
```

Comment:
A docstring's first line should be in the imperative mood (ie it should do an action instead of describing the action). It can be as simple as "Return the hero banner statement." instead of "Returns...".